portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,35 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <div id="header">
+            <h1 id="headerText">BIBI HASSAN</h1>
+            <h2>"Feet on the ground, heart on my sleeves, and head in the clouds." </h2>
+            <h2>Click here for a random fact about me:</h2>
+            <button onclick="addRandomGreeting()">Random Fact</button>
+            <div id="greeting-container"></div>
+      </div>
+      <div id=" pictures">
+          <!--photo one -->
+            <img width="300px"  height="300px" src="/images/imageOne.jpg"/>
+
+         <!--photo two -->
+            <img width="300px"  height="300px" src="/images/imageTwo.jpg"/>
+
+         <!--photo three -->
+            <img width="300px"  height="300px" src="/images/imageThree.jpg"/>
```

Comment:
Probably out of scope for this project, but another thing to keep in mind is that this only might not scale well when it comes to mobile form factors. This is usually solved using media queries: https://www.w3schools.com/cssref/css3_pr_mediaquery.asp

Not required to fix for this PR. Just an FYI :)